Makefile

```diff
@@ -83,7 +83,7 @@ export INCLUDE	:=	$(foreach dir,$(INCLUDES),-I$(CURDIR)/$(dir)) \
 			-I$(CURDIR)/lib/PowerPC_EABI_Support/Runtime/Inc \
 			-I$(CURDIR)/lib/Brawl/Include \
 			-I$(CURDIR)/lib/nw4r/include \
```

Comment:
Is this line also affected by any case sensitivity issues? It also is using lower-case.

Reply:
In this case, no because the actual path `lib/nw4r/include` is also lowercase - up to you if you want to normalize the capitalization in general.